flake.nix

```diff
@@ -207,10 +200,21 @@
         ];
       };
 
+      beherbergung-demo = nixpkgs.lib.nixosSystem (lib.mergeAttrs commonAttrs {
+        modules =
+          commonModules
+          ++ [
+            ./deployment/hosts/beherbergung-demo/configuration.nix
+            self.nixosModules.beherbergung
+            self.nixosModules.beherbergung-demo
+          ];
+      });
+
       beherbergung-lifeline = nixpkgs.lib.nixosSystem (lib.mergeAttrs commonAttrs {
```

Comment:
I touched a lot of nixos modules here.
@johannesloetzsch Please test if this does not break your lifeline setup.
` nixos-rebuild build-vm --flake '.#beherbergung-lifeline'` did not result in scary messages so far.

Reply:
ok, thanks for the hint. I will properly test when redeploying…